### PR TITLE
fix: website header

### PR DESCRIPTION
**Pull request by hgp22**
Small changes to the header layout

### CODE REVIEW

**Review by joaodiaslobo — DISMISSED** [review]
Thank you for your contribution !🙏

File: lib/integration_bee_web/components/layouts/root.html.heex [hunk 1]
```diff
@@ -11,56 +11,57 @@
...
-          >
-            Log in
-          </.link>
-        </li>
-      <% end %>
+
+  <header class="px-4 sm:px-6 lg:px-8">
+    <div class="flex items-center justify-between border-b border-zinc-100 py-3 text-sm">
+      <div class="flex items-center gap-4">
+        <a class="text-xl font-bold">UM Integration Bee</a>
```
Comment (feliciofilipe) [inline lib/integration_bee_web/components/layouts/root.html.heex:18]: This anchor tag is missing a `href`, I am assuming you wanted something like this:

```suggestion
        <a href="/" class="text-xl font-bold">UM Integration Bee</a>
```

Maybe use the `<.link/>` component

**Review by feliciofilipe — DISMISSED** [review]

**Comment by feliciofilipe** [comment]
@hgp22 LTGM 🚀  Maybe take a look at [this](https://www.conventionalcommits.org/en/v1.0.0/) commit specification in order to improve your commit and PR name 💪

**Comment by hgp22** [comment]
Managed to forget to add the _fix:_ tag to the last commit.
Hope this will do the trick x)

**Review by joaodiaslobo — APPROVED** [review]

File: lib/integration_bee_web/components/layouts/root.html.heex [hunk 2]
```diff
@@ -11,56 +11,57 @@
...
+      <li class=" text-zinc-900">
+        <%= @current_user.email %>
+      </li>
+      <li>
+        <.link href={~p"/students"} class=" text-zinc-900 font-semibold hover:text-zinc-700">
+          Students
+        </.link>
+      </li>
+      <li>
+        <.link href={~p"/users/settings"} class=" text-zinc-900 font-semibold hover:text-zinc-700">
```
Comment (ruilopesm) [inline lib/integration_bee_web/components/layouts/root.html.heex:33]:
```suggestion
        <.link navigate={~p"/users/settings"} class=" text-zinc-900 font-semibold hover:text-zinc-700">
```

File: lib/integration_bee_web/components/layouts/root.html.heex [hunk 3]
```diff
@@ -11,56 +11,57 @@
...
+          Settings
+        </.link>
+      </li>
+      <li>
+        <.link
+          href={~p"/users/log_out"}
```
Comment (ruilopesm) [inline lib/integration_bee_web/components/layouts/root.html.heex:39]:
```suggestion
          navigate={~p"/users/log_out"}
```

File: lib/integration_bee_web/components/layouts/root.html.heex [hunk 4]
```diff
@@ -11,56 +11,57 @@
...
+          method="delete"
+          class=" text-zinc-900 font-semibold hover:text-zinc-700"
+        >
+          Log out
+        </.link>
+      </li>
+    <% else %>
+      <li>
+        <.link href={~p"/users/register"} class=" text-zinc-900 font-semibold hover:text-zinc-700">
```
Comment (ruilopesm) [inline lib/integration_bee_web/components/layouts/root.html.heex:48]:
```suggestion
        <.link navigate={~p"/users/register"} class=" text-zinc-900 font-semibold hover:text-zinc-700">
```

File: lib/integration_bee_web/components/layouts/root.html.heex [hunk 5]
```diff
@@ -11,56 +11,57 @@
...
+          Register
+        </.link>
+      </li>
+
+      <li>
+        <.link href={~p"/users/log_in"} class=" text-zinc-900 font-semibold hover:text-zinc-700">
```
Comment (ruilopesm) [inline lib/integration_bee_web/components/layouts/root.html.heex:54]:
```suggestion
        <.link navigate={~p"/users/log_in"} class=" text-zinc-900 font-semibold hover:text-zinc-700">
```

**Review by ruilopesm — COMMENTED** [review]
Learn more here: https://hexdocs.pm/phoenix_live_view/live-navigation.html

**Comment by hgp22** [comment]
@ruilopesm After reading the [Live Navigation documentation](https://hexdocs.pm/phoenix_live_view/live-navigation.html) shouldn't we use `patch`?

> The "patch" operations must be used when you want to navigate to the current LiveView, simply updating the URL and the current parameters, without mounting a new LiveView.

**Comment by ruilopesm** [comment]
@hgp22 It all depends on how we want to do this. We can stick with a `patch` and change it, in the future, if needed.

**Review by ruilopesm — APPROVED** [review]

**Pull request merged**